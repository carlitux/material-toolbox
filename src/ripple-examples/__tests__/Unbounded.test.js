// @flow
import React from 'react';
import { shallow } from 'enzyme';

import UnboundedRipple from '../Unbounded';

describe('example::UnboundedRipple', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<UnboundedRipple />);
    expect(component).toMatchSnapshot();
  });
});
