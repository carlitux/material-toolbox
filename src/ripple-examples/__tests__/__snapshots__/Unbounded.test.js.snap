// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example::UnboundedRipple Should match snapshoot 1`] = `
<Inner>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';

const component = (
  <Ripple unbounded>
    <div
      tabIndex=\\"0\\"
      className=\\"ripple-demo-surface material-icons ripple-demo-surface--icon\\">
      favorite
    </div>
  </Ripple>
)"
      title="Unbounded"
    >
      <Ripple
        accent={false}
        cssOnly={false}
        disabled={false}
        primary={false}
        unbounded={true}
      >
        <div
          className="ripple-demo-surface material-icons ripple-demo-surface--icon"
          tabIndex="0"
        >
          favorite
        </div>
      </Ripple>
    </Example>
  </Cell>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';

const component = (
  <Ripple unbounded cssOnly>
    <div
      tabIndex=\\"0\\"
      className=\\"ripple-demo-surface material-icons ripple-demo-surface--icon\\">
      favorite
    </div>
  </Ripple>
)"
      title="Unbounded - CSS Only"
    >
      <Ripple
        accent={false}
        cssOnly={true}
        disabled={false}
        primary={false}
        unbounded={true}
      >
        <div
          className="ripple-demo-surface material-icons ripple-demo-surface--icon"
          tabIndex="0"
        >
          favorite
        </div>
      </Ripple>
    </Example>
  </Cell>
</Inner>
`;
